src/Layout/Connection.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { Background, Box, Text } from '../components/base';
+import { ConnectionSidebar } from '../components/ConnectionSidebar';
+import { useConnectionsProvider } from '../contexts/ConnectionsProvider';
+
+export function ConnectionLayout() {
+  const { selectedConnection } = useConnectionsProvider();
+  if (!selectedConnection) return null;
+
+  return (
+    <Background
+      css={{
+        display: 'flex'
```

Comment:
since everytime we're using the `Background` component the display is being set to flex, shouldn't we put that style on the base component?

Reply:
lgtm